src/bin/gpfdist/gpfdist.c

```diff
@@ -175,6 +176,7 @@ static struct
 	const char* ssl; /* path to certificates in case we use gpfdist with ssl */
 	int			w; /* The time waiting when used for session timeout in seconds */
 	int 		k; /* The time used to clean up sessions in seconds */
+	const char* X; /* default transformation */
 } opt = { 8080, 8080, 0, 0, 0, ".", 0, 0, -1, 5, 0, 32768, 0, 256, 0, 0, 0, 0 , 300};
```

Comment:
X isn't initialized to anything.
```suggestion
} opt = { 8080, 8080, 0, 0, 0, ".", 0, 0, -1, 5, 0, 32768, 0, 256, 0, 0, 0, 0 , 300, NULL};
```

Reply:
Thanks. But to keep the same code style it should be 0, not NULL.